WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -186,7 +202,13 @@ decisions = {
 		only_playable = yes
 		is_high_prio = yes
 		potential = {
-			religion_group = christian 
+			OR = {
+				religion_group = christian
+				religion_group = hellenic_pagan
+				religion_group = neoplatonist
+				has_romance_culture_trigger = yes
+				culture = roman
+			}
```

Comment:
Why would a Roman who isn't Christian not want to restore one of his culture's greatest spectacles. This is a purely cultural issue!